### DISCUSSION TITLE
Using Tailwind CSS as a CSS Framework

### DISCUSSION THREAD

**Discussion by joshi-kaushal** [reactplay/react-play#73] [Ideas]
Hello,

As we go ahead and get contributions from people around the globe, I think we should have a standard set for styling.

Tailwind CSS seems like a great option for me since it is widely accepted, has a minimal learning curve, and most importantly the styling is standardized. 

I can take the whole responsibility of integrating our existing app with Tailwind.

**Comment by atapas** [comment]
If we have to go ahead with an existing styling system, TailwindCSS seems an excellent choice.

I have a proposal. Can we leave the framework CSS style as is and include TailwindCSS as an option for the plays to use if they want. Some play devs may want to use just raw CSS.

How about including Tailwind as the only option outside of the regular CSS?

I am adding @nirmalkc also in the loop.

**Comment by joshi-kaushal** [comment]
Yes, plain CSS must be there. But I believe we shouldn't have more than one external styling framework.

**Reply by atapas** [reply]
That's exactly my point to. I am planning each of the plays like a micro app with their own optional package.json.. So that it can manage extra dependencies of its own. What do you think?

Btw, still complete your in-progress play with raw CSS :)

**Comment by atapas** [comment]
Hey All,

I'm pleased to confirm that, we will be able to use following Style Systmes in React Play

- Raw CSS
- SASS
- TailwindCSS

We will not encourage to use anything outside of these three at this moment.

**Reply by atapas** [reply]
@joshi-kaushal @vasantisuthar FYI.

**Reply by joshi-kaushal** [reply]
That's awesome! 

Have you integrated SASS and Tailwind with our project?

**Reply by atapas** [reply]
Opened an issue. Do you want to take up? I am looking for a quick turn around on it: https://github.com/atapas/react-play/issues/165

**Reply by atapas** [reply]
Closing this discussion as we have taken a call.

**Comment by atapas** [comment]
TailwindCSS support is in.
